hardhat.config.ts

```diff
@@ -3,6 +3,8 @@ require("@nomiclabs/hardhat-truffle5");
 require("dotenv").config();
 require("@nomiclabs/hardhat-etherscan");
 require("@nomiclabs/hardhat-web3");
+require("@nomiclabs/hardhat-vyper");
+import "@nomiclabs/hardhat-ethers";
```

Comment:
I only get "ethers from hardhat cannot be found" error in tests when it is a require not import

Reply:
import should be the way for all of them